example-config/init.nu

```diff
@@ -7,30 +7,34 @@ export def egd [...rest] {
     with-env [GIT_EXTERNAL_DIFF 'difft'] { git diff $rest }
 }
 
-# env
-export env LS_COLORS {
-    [
-       "di=01;34;2;102;217;239"
-       "or=00;40;31"
-       "mi=00;40;31"
-       "ln=00;36"
-       "ex=00;32"
-    ] | str join (char env_sep)
+# we need to export the env we create witk load-env
+# because we are `use`-ing here and not `source`-ing this file
+export-env {
+    load-env {
+            BROWSER: "firefox" 
+            CARGO_TARGET_DIR: "~/.cargo/target" 
+            EDITOR: "nvim" 
+            VISUAL: "nvim" 
+            PAGER: "less" 
+            SHELL: "~/.cargo/bin/nu" 
+            JULIA_NUM_THREADS:  nproc 
+            HOSTNAME:  (hostname | split row '.' | first | str trim)
+            SHOW_USER: true 
+            LS_COLORS: ([
+                         "di=01;34;2;102;217;239"
+                         "or=00;40;31"
+                         "mi=00;40;31"
+                         "ln=00;36"
+                         "ex=00;32" 
+                        ] | str join (char env_sep))
+    }
```

Comment:
if you were to ask me, i'd rather write that as follows, to be more consistent with the formatting in the rest of the file and more consise :yum: 
```bash
export-env {
    load-env {
        BROWSER: "firefox"
        CARGO_TARGET_DIR: "~/.cargo/target"
        EDITOR: "nvim"
        VISUAL: "nvim"
        PAGER: "less"
        SHELL: "~/.cargo/bin/nu"
        JULIA_NUM_THREADS:  nproc
        HOSTNAME:  (hostname | split row '.' | first | str trim)
        SHOW_USER: true
        LS_COLORS: ([
             "di=01;34;2;102;217;239"
             "or=00;40;31"
             "mi=00;40;31"
             "ln=00;36"
             "ex=00;32"
        ] | str join (char env_sep))
    }
}
```

Reply:
I reformatted it to move this PR along.